internal/provider/resource_ably_rule_http_cloudflare_worker_test.go

```diff
@@ -59,7 +59,8 @@ func TestAccAblyRuleCloudflareWorker(t *testing.T) {
 					"enabled",
 					"^my-channel.*",
 					"channel.message",
-					"batch",
+					// TODO: change to batch when control api not broken
```

Comment:
this comment is kinda vague, is there a github/jira ticket we can link to so that someone reading this in the future knows when it's ready to be changed back?

Reply:
Yeah I've opened #147 now. still need to chase it up with web team

Reply:
OK nice, might be worth putting that link in the comments?

Reply:
Done. Will need to be re approved.